charts/templates/configmap.yaml

```diff
@@ -22,4 +22,4 @@ data:
   DJANGO_OH_MASTERSLZ: {{ .Values.django.env.oh_masterslz }}
   DJANGO_OH_STATIC: {{ .Values.django.env.oh_static }}
   DJANGO_OH_WOWZA: {{ .Values.django.env.oh_wowza }}
-  
\ No newline at end of file
+  DJANGO_OH_PUBLIC_SITE: {{ .Values.django.env.oh_public_site }}
```

Comment:
Changing `configmap.yaml` to add a new value is appropriate,  However, there's one more change needed when doing this: the version of the Chart itself (the big-C `Chart.yaml`) also needs to be changed, when anything other than the specific values in `prod-<appname>-values.yaml` change.

Please increment `version` in `Chart.yaml`.  This typically gets bumped by a minor version, so in this case, from `0.4.0` to `0.5.0`.

There's a little more about this in our ["cookbook"](https://uclalibrary.atlassian.net/wiki/x/T075B), the final section of "Goal: Update environment variables only".